### PR TITLE
Fix wrong type hints

### PR DESCRIPTION
**Pull request by blankse**

### CODE REVIEW

File: models/Property.php [hunk 1]
```diff
@@ -25,17 +25,17 @@
...
-    protected string $type;
+    protected ?string $type = null;
 
-    protected string $ctype;
+    protected ?string $ctype = null;
 
     protected ?string $cpath = null;
 
-    protected int $cid;
+    protected ?int $cid = null;
```
Comment (lukmzig) [inline models/Property.php:38]: Hi @blankse, can this really be null? Based on the properties table cid, ctype and name are not null-able.

Comment (blankse) [reply models/Property.php:38]: Yes for new unsaved Properties:
```
$property = new Property();
$property->getCid();
```

Comment (lukmzig) [reply models/Property.php:38]: I see, thank you

**Comment by lukmzig** [comment]
Thank you @blankse

**Pull request merged**